src/utils/BrambleExtensionLoader.js

```diff
@@ -60,7 +61,8 @@ define(function (require, exports, module) {
     var extraExtensions = [
         "brackets-cdn-suggestions",    // https://github.com/szdc/brackets-cdn-suggestions
         "ImageUrlCodeHints",
-        "HTMLHinter"
+        "HTMLHinter",
+        "Autosave"
```

Comment:
So if you do `?disableExtensions=Autosave`, even though it's a default extension, we can disable it?